modules/ocf_mirrors/files/collect-mirrors-stats

```diff
@@ -17,10 +17,10 @@ from ocflib.lab.stats import humanize_bytes
 
 OCFSTATS_PWD = open(os.environ.get('OCFSTATS_PWD__FILE')).read().strip()
 MIRRORS_DATA_PATH = Path('/opt/mirrors/ftp')
-APACHE_LOG_PATH = Path('/var/log/apache2')
+APACHE_LOG_PATH = Path('/var/log/nginx')
```

Comment:
Perhaps we should rename these `APACHE_*` variables to reflect the move to Nginx.

Reply:
Good catch!